packages/react-components/react-dialog/src/components/DialogActions/renderDialogActions.tsx

```diff
@@ -1,12 +1,18 @@
-import * as React from 'react';
-import { getSlots } from '@fluentui/react-utilities';
+/* eslint-disable jsdoc/check-tag-names */
```

Comment:
Let's enable `jsxTags` in the rule config instead, https://github.com/gajus/eslint-plugin-jsdoc/blob/main/.README/rules/check-tag-names.md#jsxtags

Reply:
Follow up on this https://github.com/microsoft/fluentui/pull/27571